projects/plugins/jetpack/modules/shortcodes/facebook.php

```diff
@@ -55,12 +55,18 @@
  * @return string Facebook embed markup.
  */
 function jetpack_facebook_embed_handler( $matches, $attr, $url ) {
+	$extra_styles = 'style="background-color: #fff;"';
```

Comment:
```suggestion
	$extra_styles = 'style="background-color: #fff; display: inline-block;"';
```
There's currently a brief moment while the embed loads where the white background extends beyond the right edge of the actual embed (see ~00:03 in the attached video). I believe this change fixes that. Note that the Facebook embed eventually applies `display: inline-block` to these elements anyway—this change simply preempts that behavior.

https://github.com/user-attachments/assets/0235e898-e831-4de3-bf06-54dedd9bf928

Reply:
Sure thing! Done in 257f819bfc0508e133fa5075b0dd2b91c3968d87